test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -589,6 +589,32 @@ public void CheckToOtlpLogRecordBodyIsPopulated(bool includeFormattedMessage)
         Assert.Equal("state", otlpLogRecord.Body.StringValue);
     }
 
+    [Fact]
+    public void LogRecordBodyIsExportedWhenUsingBridgeApi()
```

Comment:
Should we add a test case to prove what happens when `otlpLogRecord.Body` is already set?

Reply:
added!